filesystem/GD.php

```diff
@@ -323,6 +323,7 @@ public function rotatePixelByPixel($angle) {
 		}
 		$rotate=imagecreatetruecolor($destWidth,$destHeight);
 		imagealphablending($rotate, false);
+        imagesavealpha($rotate, true); // to maintain PNG transparency
```

Comment:
I hate to be a pain in the butt... but could you change the indentation to tabs? SilverStripe 3 is still in the dark ages I’m afraid 😄